R/utils.R

```diff
@@ -1,11 +1,22 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
-pv_query <- function(params, reformat = TRUE, ...){
+pv_query <- function(params, reformat, ...){
+  # Run multiple queries, return as list of results
+  data_list <- lapply(as.list(params), pv_query_single, reformat = reformat)
+
+  if(reformat == TRUE){ # collapses all results into one dataframe
+    return(do.call(rbind, data_list))
+  } else {
+    return(data_list)
```

Comment:
you don't need an else here; if it gets to that statement, the statement isn't rue. If it were the function would've returned and terminated.